### PR TITLE
PR device3

### PR DESCRIPTION
**Pull request by sanchezzzhak**

### CODE REVIEW

**Comment by sanchezzzhak** [comment]
can merge?

File: regexes/device/mobiles.yml [hunk 1]
```diff
@@ -357,30 +357,47 @@ Apple:
...
 # 3Q
 3Q:
   regex: '(AC0731B|AC1024C|AC7803C|BC9710AM|EL72B|LC0720C|LC0723B|LC0725B|LC0804B|LC0808B|LC0809B|LC0810C|LC0816C|LC0901D|LC1016C|MT0724B|MT0729B|MT0729D|MT0811B|OC1020A|RC0709B|RC0710B|RC0718C|RC0719H|RC0721B|RC0722C|RC0726B|RC0734H|RC0743H|RC0817C|RC1018C|RC1019G|RC1025F|RC1301C|RC7802F|RC9711B|RC9712C|RC9716B|RC9717B|RC9724C|RC9726C|RC9727F|RC9730C|RC9731C|TS0807B|TS1013B|VM0711A|VM1017A|RC0813C|QS9719D|QS9718C|QS9715F|QS1023H|QS0815C|QS0730C|QS0728C|QS0717D|QS0716D|QS0715C|MT7801C)'
   device: 'tablet'
   model: '$1'
 
+# 4Good
+4Good:
+  regex: '(?:4Good)?(S450m [43]G|S555m 4G|S501m 3G|T700i_3G|Light A103)'
+  device: 'smartphone'
```
Comment (sgiehl) [inline regexes/device/mobiles.yml]: If I see that right all devices from 4Good start with `4Good`, so maybe we could make that detection more generic like:
```
regex: '4Good|S450m [43]G|S555m 4G|S501m 3G|T700i_3G|Light A103'
```

And add a generic model detection below:
```
- regex: '4Good[ _]([^;/]+) Build'
  model: '$1'
- regex: '4Good[ _]([^);/ ]+)'
  model: '$1'
```

File: regexes/device/mobiles.yml [hunk 2]
```diff
@@ -357,30 +357,47 @@ Apple:
...
+    - regex: 'T700i_3G'
+      model: 'T700i 3G'
+      device: 'tablet'
+    - regex: '(S(?:450m|555m|501m)[_ ][43]G|Light A103)'
+      model: '$1'
+
 # Bravis
 Bravis:
-  regex: 'Bravis|A501 Bright'
+  regex: 'Bravis|A501 Bright|NB(?:10[56]|7[145])|NP101'
```
Comment (sgiehl) [inline regexes/device/mobiles.yml]: There's also a model NB751, maybe you could add that as well.

File: regexes/device/mobiles.yml [hunk 3]
```diff
@@ -2158,6 +2183,80 @@ iBerry:
...
+  models:
+    - regex: 'NT-3506M'
+      model: 'NetTAB Mercury Quad FHD'
+
+# LYF
+LYF:
+  regex: '(LYF[ _])?LS-[45][0-9]{3}'
+  device: 'smartphone'
+  models:
+    - regex: 'LS-4004'
```
Comment (sgiehl) [inline regexes/device/mobiles.yml]: Could you also add `LS-5505` (Water F1) ?

File: regexes/device/mobiles.yml [hunk 4]
```diff
@@ -5165,6 +5264,38 @@ Fly:
...
+    - regex: 'FS551'
+      model: 'Nimbus 4'
+    - regex: 'FS454'
+      model: 'Nimbus 8'
+    - regex: 'FS452'
+      model: 'Nimbus 2'
+    - regex: 'FS451'
+      model: 'Nimbus 1'
+    - regex: 'FS405'
+      model: 'Stratus 4'
```
Comment (sgiehl) [inline regexes/device/mobiles.yml]: Could you additionally add `FS403` (Cumulus 1) and `FS401` (Stratus 1)

**Review by sgiehl — CHANGES_REQUESTED** [review]
Left a few comments for improvements. Will merge after applied

**Comment by Findus23** [comment]
Hi, I have added the necessary icons: https://github.com/piwik/piwik-icons/commit/9db9942f352ab729f4a31642f30c7d95dcdd8824

File: regexes/device/mobiles.yml (hunk 3 above)
Comment (sanchezzzhak) [inline regexes/device/mobiles.yml]: Added

File: regexes/device/mobiles.yml (hunk 4 above)
Comment (sanchezzzhak) [inline regexes/device/mobiles.yml]: Added

File: regexes/device/mobiles.yml (hunk 2 above)
Comment (sanchezzzhak) [inline regexes/device/mobiles.yml]: Added

File: regexes/device/mobiles.yml (hunk 1 above)
Comment (sanchezzzhak) [inline regexes/device/mobiles.yml]: Added

**Comment by sgiehl** [comment]
Thanks for the changes :+1:

**Review by sgiehl — APPROVED** [review]

**Pull request merged**